objcgen/processedtypes.cs

```diff
@@ -189,17 +242,20 @@ public override void ComputeSignatures (Processor p)
 	public class ProcessedProperty: ProcessedMemberBase {
 		public PropertyInfo Property { get; private set; }
 
-		public ProcessedProperty (PropertyInfo property)
+		public ProcessedProperty (PropertyInfo property, Processor processor) : base (processor)
 		{
 			Property = property;
 		}
 
-		public override void ComputeSignatures (Processor p)
+		public override void ComputeSignatures ()
 		{
 			throw new NotImplementedException ();
 		}
 
 		public override string ToString () => Property.ToString ();
+
+		public ProcessedMethod GetMethod => new ProcessedMethod (Property.GetGetMethod (), Processor);
+		public ProcessedMethod SetMethod => new ProcessedMethod (Property.GetSetMethod (), Processor);
```

Comment:
I was going to add a:

// This seems wrong....

above those lines and forgot.

You are correct. Let me redo that bit.